internal/pkg/diff/diff.go

```diff
@@ -26,6 +26,10 @@ type Differ struct {
 type ResultState int
 
 const (
+	EqualMark                    = "="
+	ChangedMark                  = "*"
```

Comment:
Taková drobnost ve wordingu, to pasivum (-d) bych použil buď všude (AddedMark, DeletedMark) nebo nikde.

Reply:
:+1:

Reply:
https://github.com/keboola/keboola-as-code/pull/309/commits/1ad1b662502f3120d1774f331d7c4425eb78ab13